src/Shared/CertificateGeneration/ProcessRunResult.cs

```diff
@@ -0,0 +1,15 @@
+using System.Text;
+
+#nullable enable
+
+namespace Microsoft.AspNetCore.Certificates.Generation
+{
+    internal record ProcessRunResult
+    {
+        public bool IsSuccess => ExitCode == 0;
+        public string CommandLine { get; init; } = "";
+        public int ExitCode { get; init; }
+        public StringBuilder? StandardOutput { get; init; }
+        public StringBuilder? StandardError { get; init; }
+    }
+}
```

Comment:
Can we remove all these layers of abstraction? It quickly adds to the maintainability cost and doesn't bring much benefits.

I don't think we use ProcessRunner in any other area of this tool and that hasn't been a problem so far, I would prefer if we stick to what we are doing elsewhere (like Mac OS) for consistency.